docs/reference/output.md

```diff
@@ -74,7 +74,10 @@ Output a function for each method in the conversion interface.
 
 `output:package [PACKAGE][:NAME]` can be defined as
```

Comment:
Can you adjust docs/guide/output-same-package.md, the is written that the full package path has to be defined, this isn't required anymore.